about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2021-07-16 17:00:00 -0700
+---
+
+## Applicability
+{: .text-left}
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+In order to reduce the friction of adding data to the SGID, by submitting a [Porter issue](https://github.com/agrc/porter/issues) to introduce your data to the SGID, you consent to this Data Sharing Agreement. In cases where UGRC submits the Porter issue on your behalf, UGRC will contact you via email to get your consent to this agreement.
```

Comment:
should this porter link (eventually) open a new issue using a data sharing-specific template?  maybe the porter template makes mention of this page as a way of saying you agree to these terms: specifically, the "The steward will ... ".

Reply:
Yeah, I am planning on making those edits once this gets merged.